test/unit/ajax-test.js

```diff
@@ -221,6 +224,8 @@ define([
                 done();
             }
         });
+        server.respond();
```

Comment:
This will create the same response in every browser. The purpose of this test is to verify xhr error handling across platforms.